### PR TITLE
Add the chemistry bag as the default belt for the chemist

### PR DESCRIPTION
**Pull request by joelhed**
## About the PR

This PR adds the *chemistry bag* as the default belt for the chemist.

Currently, chemists spawn with an empty medical belt, which is usable as generic storage, but isn't really specific to the role, such as for engineers with their utility belt, medical doctors with ointment and bruise packs, botanists with the basic botany tools, security officers with ... sec stuff, etc.  This new chemistry bag would be the in the chemist's belt slot by default, and is quite similar to the medical belt, but can only hold *chemistry products*, namely pills, pill canisters, bottles, and syringes.  In turn, it has a storage space of 45, 5 more than the medical belt. The bag also has areaInsert and the Dumpable component enabled, which makes it easy to quickly pick up and drop products wherever necessary.  Along with the already existing *"Transfer contents"* right-click verb, this bag would serve as a small but valuable QoL tool for the chemistry department.

Additionally, this bag is essential for the eventual implementation of the SmartFridge (#5071), as it would serve as the intermediate storage space for *products* between the ChemMaster and the fridge, separate from the backpack.  Having this organisation by default would make life easier for many chemists, both now and in the future.  And if you don't like it, you can always opt out and grab yourself an empty medical belt from the ChemDrobe.

### Previous concerns 

The original implementation of this bag was made by @reapsnI in #11540, but was closed/denied at that time by @moonheart08 for being too overpowered and excessive.  The capacity is adjusted to hold 45 individual pills, i.e. the contents of 5 full pill canisters, or 9 pill canisters directly, giving it a max capacity of 81 pills, which is a reasonable amount compared to medical belt's current capacity.  I believe that with this storage capacity, the bag would be more balanced and less excessive than the previous PR, and would serve as a valuable addition to the chemistry role. 

With this size, it would hardly make medkits and medical belts worth any less, as @DummeHytteOst implied, and it would still be useful as an intermediate storage location for pills etc., even though pill bottles already have the Dumpable feature.  @EmoGarbage404 also raised some concerns that its introduction would go in the opposite direction from the planned [Chemistry redesign](https://github.com/space-wizards/space-station-14/milestone/3), which is aiming to make chemistry more resource-strict.  However, the chemistry department will still produce some kind of output, although probably not in the same magnitude as it is currently, so the bag would still have a purpose in this future scenario.

There are some interesting ideas floating around that address similar QoL improvements as this, such as [a production pill bottle](https://discord.com/channels/310555209753690112/1059732915421991002) by @QuietlyWhisper.  This PR wouldn't conflict with such an addition, but would simply make working chemistry a little bit more varied.

Seeing as this addition was violently struck down the last time it was introduced, I'm open to tweaking this further so that it fits the game better.  Or if there is some fundamental issue with the core idea of a chem bag in the game, we can defeat the idea once and for all.

Again, most of the credit for this goes to @reapsnI.  All I've done is tweak their implementation slightly.

**Media**

examined-on-floor

appearance

chemdrobe

a gif where the contents of a pill canister is transferred into the chemistry bag and the bag is subsequently dumped on a counter

- [x] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

**Changelog**

:cl: reapsnI and joelhed
- add: Added the chemistry bag as the default belt for the chemist!

### CODE REVIEW

**Comment by arealbeba** [comment]
very cool

**Comment by reapsnI** [comment]
TBH, that was my main idea, to create an item wich will be a intermediate storage item, between chemmaster and a smartfridge. Very cool that you made a change to it. Requseting a review will be good as well.

File: Resources/Textures/Objects/Specific/Chemistry/chem_bag.rsi/meta.json [hunk 1]
```diff
@@ -0,0 +1,18 @@
+{
+  "version": 1,
+  "license": "CC-BY-SA-3.0",
+  "copyright": "Taken from paradise station https://github.com/ParadiseSS13/Paradise and rescaled by reapsnI",
```
Comment (AJCM-git) [inline Resources/Textures/Objects/Specific/Chemistry/chem_bag.rsi/meta.json:4]: link the commit you took it (and the dmi itself in said commit if possible)

**Comment by AJCM-git** [comment]
@moonheart08 opinion on this?

File: Resources/Textures/Objects/Specific/Chemistry/chem_bag.rsi/meta.json (hunk 1 above)
Comment (joelhed) [reply Resources/Textures/Objects/Specific/Chemistry/chem_bag.rsi/meta.json:4]: @reapsnI Could you look that up? Or do you roughly know what date you downloaded it from paradise?

Comment (joelhed) [reply Resources/Textures/Objects/Specific/Chemistry/chem_bag.rsi/meta.json:4]: Anyway, looked up the last commit in Paradise before @reapsnI made his commit as a reasonable estimate. Would a good string be `"Taken from ParadiseSS13/Paradise at https://github.com/ParadiseSS13/Paradise/blob/ebf9a7b5e89f5523c5e1c3d2692bf54dac322cae/icons/obj/chemical.dmi and rescaled by reapsnI"`? @AJCM-git

**Comment by mirrorcult** [comment]
i think this is fine

**Comment by reapsnI** [comment]
@joelhed 🤝

**Pull request merged**